<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesManagement</class>
 <widget class="QMainWindow" name="SalesManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-20</y>
      <width>621</width>
      <height>661</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 62, 62);</string>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="TableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>561</width>
      <height>321</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>561</width>
      <height>0</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(182, 182, 136);
background-color: rgb(148, 148, 148);</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>SoldBill</string>
     </property>
     <property name="font">
      <font>
       <family>Imprint MT Shadow</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignTop</set>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
     <property name="icon">
      <iconset theme="applications-games">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PurchasedBill</string>
     </property>
     <property name="font">
      <font>
       <family>Imprint MT Shadow</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="font">
      <font>
       <family>Imprint MT Shadow</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="font">
      <font>
       <family>Imprint MT Shadow</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="SaleLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 252, 188);
background-color: rgb(203, 203, 151);</string>
    </property>
   </widget>
   <widget class="QLabel" name="SaleLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;SoldBill&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="BuyLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;BoughtBill&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="SmLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 62, 62);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#e2e2a8;&quot;&gt;Sales Management&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="BuyLineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(195, 195, 145);</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="AddButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>405</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Images-Icons/icons8-add-60.png</normaloff>../Images-Icons/icons8-add-60.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../StoreManagement/Images-Icons/icons8-back-arrow-100.png</normaloff>../StoreManagement/Images-Icons/icons8-back-arrow-100.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UpdatePB">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>450</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;
color: rgb(255, 255, 255);
border:2px;
border-radius:20px;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../StoreManagement/Images-Icons/qtt.qrc"/>
 </resources>
 <connections/>
</ui>
